### ISSUE TITLE
Invalid array definitions in JSON schemas

### ISSUE THREAD

**Issue by mxr576** [asyncapi/spec-json-schemas#490]
#### Describe the bug
I am building a validator in PHP based on https://github.com/justinrainbow/json-schema and that package and PHP in general turned out to be more strict regarding schema validation than Javascript. Therefore I have a strong gut feeling that the issue that the above-mentioned library reports when it tries to validate https://github.com/asyncapi/spec/blob/v2.6.0/examples/streetlights-kafka.yml on https://github.com/asyncapi/spec-json-schemas/blob/v6.4.0/schemas/2.6.0-without-%24id.json are in the schema definitions and not bugs in the validator.

By making the following changes to JSON schema, these reported issue goes away.

![image](https://github.com/asyncapi/spec-json-schemas/assets/1755573/a81069e6-5e7a-4ad7-b648-b25aae6e9daf)

#### How to Reproduce
Steps to reproduce the issue. Attach all resources that can help us understand the issue:

- Screenshots
- Link to GitHub repository with project that has issues
- Files that can help reproduce the issue, like your AsyncAPI file, either paste inside the issue in a code block or in a sharable [gist](https://gist.github.com/)

**Comment by jonaslagoni** [comment]
@mxr576 the library does not support JSON Schema draft 7 which the documents are written in, you need to down grade the schema before using the validator.

**Comment by mxr576** [comment]
@jonaslagoni you are absolutely right, after your message I have also found https://github.com/justinrainbow/json-schema/issues/576... sorry for the noise here.

(To my excuse, I have also checked several JSON Schema examples, docs to see if this should be valid or not...)
